### DISCUSSION TITLE
How should installers without silent installation switches be handled?

### DISCUSSION THREAD

**Discussion by ccnly** [microsoft/winget-pkgs#16233] [Q&A] [answered]
Can installation files without any silent installation options be submitted as manifests?

_Example: [Logitech G HUB](https://download01.logi.com/web/ftp/pub/techsupport/gaming/lghub_installer.exe) is provided as an x64 .exe file with no known or published switches at all._

**Comment by denelon** [comment, marked as answer]
https://github.com/microsoft/winget-cli/issues/910
